src/hypervisor.c

```diff
@@ -107,6 +108,43 @@ cc_oci_expand_net_device_cmdline(struct cc_oci_config *config, guint index) {
 	return g_strdup("");
 }
 
+/*
+ *
+ *
+ */
+#define DPDK_CMDLINE_OBJ "memory-backend-file,id=dpdkmem,size=2048M,mem-path=/dev/hugepages,share=on,prealloc=on"
```

Comment:
the 2048 is linked to the actual CC container mem size right. 

_But_ that is hardcoded in the hypervisor args.

Longer term we should make that param variable in the hypervisor args as there will be other cases where VM memory size maps to other QEMU parameters.